pkg/components/modelserver/modelserver.go

```diff
@@ -168,14 +165,24 @@ func NewService(k *v1alpha1.Kepler) *corev1.Service {
 			Selector:  labels,
 			Ports: []corev1.ServicePort{{
 				Name:       "http",
-				Port:       exporterPort,
+				Port:       int32(port),
 				TargetPort: intstr.FromString("http"),
 			}},
 		},
 	}
 }
 
-func NewConfigMap(d components.Detail, k *v1alpha1.Kepler) *corev1.ConfigMap {
+func GetModelServerConfigForClient(ms *v1alpha1.ModelServerSpec) map[string]string {
```

Comment:
```suggestion
func ModelServerConfigForClient(ms v1alpha1.ModelServerSpec) k8s.StringMap {
```
Keeping it k8s.StringMap has the advantage that we can merge 2 or more maps.